### ISSUE TITLE
bug: Redesign time share button

### ISSUE THREAD

**Issue by GKid94** [ReVanced/revanced-patches#3210]
### Bug description

The share time button needs a design change because it's hard to see what the button is for
![IMG_20231028_220743](https://github.com/ReVanced/revanced-patches/assets/18757054/67faf41a-eecf-49dc-9c7e-135bb3df3f18)

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by oSumAtrIX** [comment]
I agree with @LisoUseInAIKyrios, though the user may be too hesitant to click on things they don't know, so it may be considerable to change it. Please make suggestions

**Comment by GKid94** [comment]
> When the button is pressed It shows a toast. The user will figure it out after they try it.

Toast?

**Comment by KobeW50** [comment]
> Toast?

![Screenshot_20231028_201803.jpg](https://github.com/ReVanced/revanced-patches/assets/84587632/6e873ea3-df37-4e1e-967e-c4bf5b89c0f7)

**Comment by GKid94** [comment]
> > Toast?
> 
> ![Screenshot_20231028_201803.jpg](https://user-images.githubusercontent.com/84587632/278843002-6e873ea3-df37-4e1e-967e-c4bf5b89c0f7.jpg)

Oh ok

**Comment by MondayNitro** [comment]
![Screenshot_20231030-141150_Trebuchet](https://github.com/ReVanced/revanced-patches/assets/87489540/fb9cc54f-0739-404d-930e-f204efe4024a)
The clock is better visible in inotia's icon

**Comment by GKid94** [comment]
> ![Screenshot_20231030-141150_Trebuchet](https://user-images.githubusercontent.com/87489540/278989194-fb9cc54f-0739-404d-930e-f204efe4024a.png) The clock is better visible in inotia's icon

This looks better 👍
